src/hidra/receiver/plugins/asapo_producer.py

```diff
@@ -46,7 +46,7 @@
         n_threads: 1
         user_config_path: 'path/to/conf'
         file_regex: '.*/(?P<data_source>.*)/scan_(?P<scan_id>.*)/'
-                    '(?P<file_idx_in_scan>.*).tif'
+                    '(?P<file_idx_in_scan>.*)_(?P<file_idx_offset>.*).tif'
```

Comment:
I would change the order of index and offset in this example, because I guess "normally" the index is written by the detector at the end of the filename.

Reply:
Done.